Kconfig.zephyr

```
@@ -292,6 +292,17 @@ config LINKER_LAST_SECTION_ID_PATTERN
 	  be used.
 	  The size of the pattern must not exceed 4 bytes.
 
+config LINKER_USE_RELAX
+	bool "Use --relax Linker option"
+	default y
+	help
+	  This option performs global optimizations that become possible when the linker resolves
+	  addressing in the program, such as relaxing address modes and synthesizing new
+	  instructions in the output object file. On platforms where this is not supported,
+	  `--relax' is accepted, but ignored.
```

Minor nit, as Zephyr is working towards better 3rd party toolchain support then placing ld/lld refs here is nice.
For example:
```suggestion
	  instructions in the output object file.
	  For ld and lld, this enables `--relax`. On platforms where this is not supported,
	  `--relax' is accepted, but ignored.
```